sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1937,7 +1960,7 @@ def task_stop(self):
         self.task_stopping_event.set()
         os.kill(self.task_process.pid, signal.SIGKILL)
 
-    def on_port_config_change(self , port_change_event):
+    def on_port_config_change(self, stopping_event, port_change_event):
```

Comment:
@abohanyang 
Per the description, "on_port_config_change takes three arguments, not two. This issue is found when the port breakout mode changes"
Have a Q: 'stopping event' is passed as an argument but not used/consumed?
How is this related to port breakout changes? what's missing/expected?
the platform/board you are finding & validating 'port breakout change' is PHY-less (i.e. does not have Ext PHY (gearbox))?

Reply:
Yes, this stopping_event is passed and is not used at all. I actually don't know why we have this stopping_event. I guess this stopping_event will be used to handle some error cases.

My understanding is that this on_port_config_change function is triggered during the port breakout change since the old port is removed (PORT_REMOVE) from the port_mapping and the new port is added (PORT_ADD) to the port_mapping. 

My platform is a PHY-less platform. DNX NIF is connected to the OSFP/QSFP-DD front panel directly.

Reply:
> Yes, this stopping_event is passed and is not used at all. I actually don't know why we have this stopping_event. I guess this stopping_event will be used to handle some error cases.
> 

If unclear, can we avoid adding this change to this PR?

> My understanding is that this on_port_config_change function is triggered during the port breakout change since the old port is removed (PORT_REMOVE) from the port_mapping and the new port is added (PORT_ADD) to the port_mapping.
> 

Would you be validating port breakout as part of this PR? If not, then IMO, this should be handled in separate PR

> My platform is a PHY-less platform. DNX NIF is connected to the OSFP/QSFP-DD front panel directly.
Thanks for clarifying.

Reply:
Okay. I can create a separate PR to fix this.